### PR TITLE
Fixes #21619: Wrong name use for campaign in report (should not canonify)

### PR DESCRIPTION
**Pull request by VinceMacBuche**
https://issues.rudder.io/issues/21619

### CODE REVIEW

File: techniques/applications/systemUpdateCampaign/1.0/systemUpdateCampaign.st [hunk 1]
```diff
@@ -37,7 +37,7 @@ bundle agent rudder_system_update_campaign_&RudderUniqueID& {
       "pass1"     expression => "any";
 
     pass2::
-      "call"      expression => usemodule("system_update.py", "--package_manager=${package_module_knowledge.platform_default} --campaign_id=${campaign_id} --start=${start} --end=${end} --node_id=${g.uuid} --agent_schedule=${g.agent_run_interval} ${reboot_opt}"),
+      "call"      expression => usemodule("system_update.py", "--package_manager=${package_module_knowledge.platform_default} --campaign_id=${campaign_name} --start=${start} --end=${end} --node_id=${g.uuid} --agent_schedule=${g.agent_run_interval} ${reboot_opt}"),
```
Comment (amousset) [inline techniques/applications/systemUpdateCampaign/1.0/systemUpdateCampaign.st:40]: isn't the id suppoed to be a uuid or a canonified string? If not we nee to modify the script too to canonify it before using it in filenames.

Comment (VinceMacBuche) [reply techniques/applications/systemUpdateCampaign/1.0/systemUpdateCampaign.st:40]: an uuid

Comment (amousset) [reply techniques/applications/systemUpdateCampaign/1.0/systemUpdateCampaign.st:40]: with _tiret du six_?

Comment (VinceMacBuche) [reply techniques/applications/systemUpdateCampaign/1.0/systemUpdateCampaign.st:40]: the famous "tiret du six" aka a dash

Comment (VinceMacBuche) [reply techniques/applications/systemUpdateCampaign/1.0/systemUpdateCampaign.st:40]: I kep the canonfied value where it should (in cfengine classes), I checked in the module that it was just used as a filename

**Pull request merged**